receiver/prometheusreceiver/internal/otlp_transaction.go

```diff
@@ -157,6 +167,20 @@ func (t *transaction) Rollback() error {
 	return nil
 }
 
+func (t *transaction) AddTargetInfo(labels labels.Labels) error {
+	attrs := t.nodeResource.Attributes()
+
+	for _, lbl := range labels {
+		if lbl.Name == model.JobLabel || lbl.Name == model.InstanceLabel || lbl.Name == model.MetricNameLabel {
```

Comment:
Does this include scrape metadata labels (e.g. __meta_kubernetes_pod...)

Reply:
Labels starting with `__` are all dropped before the storage layer.

> Labels starting with __ will be removed from the label set after target relabeling is completed.

From: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config

https://github.com/prometheus/prometheus/blob/c511d26dd013d28ab3ff3a403744e51b81247414/scrape/scrape.go#L648

and

https://github.com/prometheus/prometheus/blob/c511d26dd013d28ab3ff3a403744e51b81247414/scrape/target.go#L177-L179

Reply:
Got it, thanks.  I was misremembering how I did #9416